.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Motivation
```

Comment:
I don't think this template is actually very useful. @dcgoss your choice whether to keep or remove it.